### PR TITLE
Removes a bunch of tooling messes and stuff in the project root.

### PR DESCRIPTION
**Pull request by PJB3005**
PLEASE make sure I'm available when this is merged so I can tell MoMMI to use the correct path for `ss13_genchangelog.py`

### CODE REVIEW

**Review by N3X15 — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,36 +0,0 @@
-before_script:
-  - $wololo
```
Comment (gbasood) [inline .gitlab-ci.yml:2]: why was this ever here

Comment (PJB3005) [reply .gitlab-ci.yml:2]: I don't know you made this file didn't you?

Comment (gbasood) [reply .gitlab-ci.yml:2]: I sure as fuck didn't add that line
https://gitlab.com/d3athrow/vgstation/commit/31d8ae15a2834c7cdf61a5dc63c275191f926d00
Pomf did for arcane reasons

Comment (Probe1) [reply .gitlab-ci.yml:2]: He was converting it to the blue team.

**Comment by Probe1** [comment]
wololo_by_shabazik-daihrjx

**Review by optimumtact — APPROVED** [review]

**Pull request merged**